mountpoint-s3/src/inode.rs

```diff
@@ -238,7 +238,27 @@ impl Superblock {
                 .update_or_insert(parent_ino, name, stat.clone(), InodeKind::File)?;
             Ok(LookedUp { inode, stat })
         } else {
-            Err(InodeError::FileDoesNotExist)
```

Comment:
Should we also add this case in the comments at line 233, otherwise comment can be misleading?

Reply:
Updated, I think we can say on line 233 that the object doesn't exist remotely.